src/components/ComingSoon.tsx

```diff
@@ -6,7 +6,7 @@ import { useStringGetter } from '@/hooks/useStringGetter';
 
 import { layoutMixins } from '@/styles/layoutMixins';
 
-export const ComingSoon = () => {
```

Comment:
if we remove `export` here, we may as well just remove the component as a whole.

Reply:
it's used below